### PR TITLE
Bcrypt - hash and verify password

### PR DESCRIPTION
**Pull request by AlexeyT74**
1. Added bcrypt 
2. bcrypt.test
3. Small part of Auth

### CODE REVIEW

File: classes/fullstack/project1/server/src/auth/index.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { hashPassword, verifyPassword } from '../utils/bcrypt'
```
Comment (Darkmift) [inline classes/fullstack/project1/server/src/auth/index.ts:1]: this is not part of your issue scope please remove this file

File: classes/fullstack/project1/server/src/utils/bcrypt.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import bcrypt from 'bcryptjs'
```
Comment (Darkmift) [inline classes/fullstack/project1/server/src/utils/bcrypt.ts:1]: clear and well documented!

File: classes/fullstack/project1/server/src/utils/bcrypt.unit-test.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { hashPassword, verifyPassword } from './bcrypt'
```
Comment (Darkmift) [inline classes/fullstack/project1/server/src/utils/bcrypt.unit-test.ts:1]: Good test coverage

**Review by Darkmift — CHANGES_REQUESTED** [review]

File: classes/fullstack/project1/server/src/auth/index.ts (hunk 1 above)
Comment (AlexeyT74) [reply classes/fullstack/project1/server/src/auth/index.ts]: ok. removed

File: classes/fullstack/project1/server/src/utils/bcrypt.ts (hunk 2 above)
Comment (AlexeyT74) [reply classes/fullstack/project1/server/src/utils/bcrypt.ts:1]: removed

File: classes/fullstack/project1/server/package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bcrypt": "^5.1.1",
+    "bcryptjs": "^2.4.3",
```
Comment (Darkmift) [inline classes/fullstack/project1/server/package.json:23]: good catch!

**Review by Darkmift — APPROVED** [review]

**Review by Darkmift — APPROVED** [review]

**Comment by Darkmift** [comment]
good work!

**Pull request merged**